libraries/AP_BattMonitor/AP_BattMonitor.cpp

```diff
@@ -149,6 +150,13 @@ AP_BattMonitor::init()
                                                                     hal.i2c_mgr->get_device(_params[instance]._i2c_bus, AP_BATTMONITOR_SMBUS_I2C_ADDR,
                                                                                             100000, true, 20));
                 break;
+            case Type::Rotoye:
+                // For the PixHawk4, the i2c_bus paramter did not work, had to hard-code this to 3 for testing
```

Comment:
Is this comment correct?

Reply:
Yes, in terms of our testing on Pixhawk 4, this is still an issue.

Reply:
I don't think this should be an inline comment as it will go stale quickly and be unclear if it's correct or not. We should have an issue open if this is a problem.

Reply:
that doesn't make sense. Can you add a hal.console->printf() for _i2c_bus here?

Reply:
Please remove this comment, then we can get this merged.

Reply:
we can just remove the comment

Reply:
Removed the comment